apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml

```diff
@@ -99,10 +99,16 @@ spec:
                     title: RPX-PR
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api|.HMCTS_Nightly.*CDM.*).+\/master
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_Nightly.*CDM.*\/(?!.*ccd-admin-web-api).+\/master
+                    name: CDM-Nightly
+                    recurse: true
+                    title: CDM Nightly Builds Dashboard
                 - buildMonitor:
                     includeRegex: >-
                       ^HMCTS_.*CDM.*\/(.*ac-manage-case-assignment.*PR-272|.*ccd-admin-web.*PR-325|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260|.*ccd-data-store-api.*PR-1260|.*ccd-definition-store-api.*PR-575|.*ccd-user-profile-api.*PR-399|.*cpo-case-payment-orders-api.*PR-54|.*ccd-api-gateway.*PR-353|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260|.*ccd-case-management-web.*PR-1043)
```

Comment:
Could you please also check the fullness and correctness of develop builds? Some apps with a develop branch doesn't appear under that dashboard. The PR numbers for develop into master of those repos may have just changed to a new PR recently. CCD Case Management Web is one of those, for example.